lxd/operations.go

```diff
@@ -38,6 +38,12 @@ var operationWait = APIEndpoint{
 	Get: APIEndpointAction{Handler: operationWaitGet, AccessHandler: allowAuthenticated},
 }
 
+var operationWaitSecret = APIEndpoint{
+	Path: "operations/{id}/wait/secret/{secret}",
+
+	Get: APIEndpointAction{Handler: operationWaitGet, AllowUntrusted: true},
+}
```

Comment:
See comment from other commit, this shouldn't be a separate route, we just need the normal `operations/{id}/wait` to use FormValue("secret") to get the `?secret=BLAH` part of the URL. `operationWebsocketGet` has that logic already.